packages/browser-wallet/src/popup/pages/Account/Account.tsx

```diff
@@ -35,7 +38,18 @@ function Account({
     return (
         <div className="flex-column justify-space-between align-center h-full relative">
             <div className="account-page__content">
-                {accounts.length === 0 && <div>{t('noAccounts')}</div>}
+                {accounts.length === 0 && (
+                    <div className="flex-column align-center h-full">
+                        <p className="m-t-20 m-h-40">{t('noAccounts')}</p>
+                        <Button
+                            className="m-b-40 m-t-auto"
+                            width="wide"
+                            onClick={() => nav(absoluteRoutes.home.identities.add.path)}
```

Comment:
This navigates to identities. If you have no identity, then the text could show that and have a button for that, but the create account button should not do that.

Reply:
That was a brainfart, but I discussed with Jens and for now it is fine to link to add accounts, because that links to add identities if there are no identities (and this is not a "normal" flow)